benchmarks/distance_matrix/config.json

```diff
@@ -0,0 +1,73 @@
+{
+  "file": "SUSY-{size}.h5",
+  "dataset": "data",
+
+  "trials": 10,
+
+  "benchmarks": {
+    "dask-auto-cpu": {
+      "template": "dask",
+      "nodes": [2, 3, 5, 9, 15],
+      "tasks": [1, 1, 1, 1, 1],
+      "size": {
+        "strong": "40k",
+        "weak": [25820, 36515, 51640, 73030, 100000]
+      }
+    },
+
+    "dask-chunks-cpu": {
```

Comment:
For the Cdist benchmarks, the Chunking sizes for dasked-tuned also varied with the number of nodes/dataset size: = `` size(dataset) // number of nodes``